scripts/fuzz_shell.js

```diff
@@ -175,7 +175,15 @@ var imports = {
 imports = Asyncify.instrumentImports(imports);
 
 // Create the wasm.
-var instance = new WebAssembly.Instance(new WebAssembly.Module(binary), imports);
+var module = new WebAssembly.Module(binary);
+
+var instance;
+try {
+  instance = new WebAssembly.Instance(module, imports);
+} catch (e) {
+  console.log('Failed to instantiate module');
+  quit();
```

Comment:
Hmm, I'm worried about suppressing an error here... that seems unsafe, even if it helps a particular fuzzer situation. I guess the logging would make it obvious to anyone reading the output, but a zero error code could confuse tools.

Reply:
I guess the definition of what is an error depends on the specific use case, but I don't think we want to consider traps during instantiation to be errors in any of our fuzzing use cases, just like we don't consider traps after instantiation to be errors.

Reply:
Ah, very good point! Yes, traps during instantiation should not be special.

In that case please make the logging here match the exception logging at the bottom of the file, see other comment.